Sources/searchable-collection-vc/SearchableCollectionViewLayout.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SearchableCollectionViewLayout.swift
+//  Demo
+//
+//  Created by Yuliia Veresklia on 20/04/2017.
+//
+//
+
+import Foundation
+import UIKit
+
+class SearchableCollectionViewLayout: UICollectionViewFlowLayout {
+    
+    private var searchBackgroundColor = SearchBarView.defaultSearchBackgroundColor
+    
+    override init() {
+        super.init()
+        register(SearchBarView.self, forDecorationViewOfKind: SearchBarView.kind)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func layoutAttributesForElements(in rect: CGRect) -> [UICollectionViewLayoutAttributes]? {
+        var attributes = super.layoutAttributesForElements(in: rect)
+        
```

Comment:
@yuliaveres Or you could do it like this:
`let attributes = super.layoutAttributesForElements(in: view.bounds)?.flatMap({ $0.copy() as? UICollectionViewLayoutAttributes })`